main.go

```diff
@@ -22,6 +22,11 @@ func main() {
 		// plugin args
 		//
 
+		cli.BoolTFlag{
```

Comment:
Are we concerned that you added this flag as a `BoolTFlag` type?

My concern here is that since the flag would hold a [default value of true](https://github.com/urfave/cli/blob/master/flag_generated.go#L59), then the plugin would perform a dry run by default and that could be unexpected behavior for consumers.

I'd rather, it be added as a `BoolFlag` so the dry run functionality would have to be explicitly specified. This is how the [docker plugin](https://github.com/drone-plugins/drone-docker/blob/master/cmd/drone-docker/main.go#L28-L32) works as well.